src/assests/skeltons/ListingSkelton.component.tsx

```diff
@@ -2,7 +2,7 @@ import {
  Skeleton, Layout, PageHeader, Row, Col,
 } from 'antd';
 import React, { FC, memo } from 'react';
-import TableSkelton from 'assests/skeltons/TableSkelton.component';
```

Comment:
relative imports are not working in storybook

Reply:
you have fixed imports now, right ?

Reply:
yes